totest-manager.py

```diff
@@ -811,6 +811,34 @@ def get_current_snapshot(self):
         return self.iso_build_version(self.project + ':ToTest', self.main_products[0])
 
 
+class ToTest151ARM(ToTestBaseNew):
+    main_products = [
+        '000product:openSUSE-cd-mini-aarch64',
+        '000product:openSUSE-dvd5-dvd-aarch64',
+    ]
+
+    ftp_products = ['000product:openSUSE-ftp-ftp-aarch64',
+                    '000product:openSUSE-ftp-ftp-armv7hl',
+                    ]
+
+    livecd_products = []
+
+    # Leap 15.1 ARM still need to update snapshot
+    set_snapshot_number = True
+
+    # product_repo openqa_group jobs_num values are specific to aarch64
+    # TODO: How to handle the other entries of main_products ?
+
+    def openqa_group(self):
+        return 'openSUSE Leap 15.1 AArch64'
```

Comment:
done

Reply:
@nilxam the reason why we went for not mentioning the minor version in the openQA job group is less work for each minor version. This might be step towards also having less minor versions handled explicitly in OBS eventually